tests/testthat/test-result_data_frame.R

```diff
@@ -583,10 +585,14 @@ test_that("make_ard works when printed format differs from cell values", {
 
   expect_warning(
     out <- as_result_df(test_out, make_ard = TRUE, verbose = TRUE),
-    "Found 9 cell"
+    paste0(
+      "Found 1 cell values that differ from printed values. ",
+      "This is possibly related to conditional formatting. ",
+      "\\\nThe following row names were modified: Mean \\(SD\\), Mean \\(SD\\)\\\n"
```

Comment:
one of the issues concerned this specific output. Hence, I wrote it all. It was giving all the lines, which was not exact (lengths_keep_nuls solves this)